.huskyrc

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": {
+    "pre-commit": "lint-staged",
```

Comment:
in pre-commit we could test the related files with jest

Reply:
i dont think i got what you mean. example?

Reply:
by default jest tests only files that have changed

Reply:
and all related files

Reply:
or maybe it was default for react-scripts test:

see here: https://github.com/City-of-Helsinki/palvelutarjotin-ui/blob/develop/package.json

Reply:
` "test:staged": "SASS_PATH=./src/assets/styles jest --findRelatedTests",`